### DISCUSSION TITLE
Does the Z-Offset tool actually work?

### DISCUSSION THREAD

**Discussion by cmtekwerx** [Jyers/Marlin#267] [Q&A] [answered]
Hi,

Yes, I'm a noob. After working with the stock firmware and Smith3D for a while, I decided to try out this one. I have an Ender 3 V2, with a BLTouch. I downloaded the EV2-UBL-BLTouch-10x10-v4-2-2.bin file. 

The 10x10 probing is nice if not time consuming, but it shouldn't need done all the time.

Checking the Z position after an Autohome, the Z is +10 over the bed, as I would expect from having the BLTouch installed. 

In the stock firmware, I would do the following:

1. Use the the control panel to move Z down to 0 (from 10)
2. Use paper to determine the the Z offset, and maybe fine tune if needed during a print

In Smith3D - more or less the same, but with a very nice and functional Z Offset Tool.

In the version I just installed - I have no freaking clue what to do...

1. Manually moving the the Z from 10 to 0 using the control panel, does move the Z down to just above the bed
2. Entering the Z Offset menu, selecting Live Adjustment causes the Z axis to move back to 10 above the bed 
3. I know my Z Offset is right around -2.8 (at least it was in Smith3D), and it's impossible to dial it down that far using the Z Offset menu.
4. It's tough to do the paper test with no Live Adjustment, let alone when Rehoming Z causes it to go back to the default +10 position.

What am I missing? I'm guessing I am missing something - probably really simple...

**Comment by cmtekwerx** [comment, marked as answer]
This issue appears either in only the UBL for the BLTouch 10x10 or I just didn't know what I was doing.

I downloaded the 5x5 version of the BLTouch (non-UBL) and the Z Offset worked as expected.

**Reply by Jyers** [reply]
The Z offset tool should function no differently between different skews, but I will test it with UBL to make sure there isn't an issue.

**Reply by cmtekwerx** [reply]
Ok. I can say the 5X5 I'm using has been rock solid so far.
